lib/kapnismology/smoke_test_run.rb

```diff
@@ -0,0 +1,33 @@
+require 'json'
+
+module Kapnismology
+  # A summary of a run containing smoke test evaluations
+  class SmokeTestRun
+
+    PROFILE_URL = 'http://tbd.mdsol.com'.freeze
+
+    def initialize(evaluations)
+      @evaluations = evaluations
+    end
+
+    def status
+      @evaluations.passed? ? 200 : 503
+    end
+
+    def results(request_url)
```

Comment:
this I think will need always to get a `.to_json` afterwards. Instead of that, call it `render` or something like that and do inside the method